middleware/middleware.go

```diff
@@ -74,7 +74,7 @@ func Version(v string) func(http.Handler) http.Handler {
 func Logging(logger logrus.FieldLogger) func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			reqID := uuid.NewString()
+			reqID := uuid.New().String()
```

Comment:
Why is this needed

Reply:
I believe its just because of the uuid version. Looks like `NewString` is not available.

Reply:
Indeed, the `NewString` is not available and it was making tests to fail.